src/ui-lib/sass/03-elements/_input.scss

```diff
@@ -0,0 +1,7 @@
+input[type=text],
+input[type=password] {
```

Comment:
Should we add other textbox style inputs here: `email`, `url`, `tel`... etc. See: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input

In fact, it may be easier to set these defaults on any `input` and then override for the handful of exceptions like `radio` and `checkbox` (actually, they might be the only ones that don't render a text box).

Reply:
I've gone applying styling to all `input` types and have used `:not` selectors to exclude `radio` and `checkbox`. We may want to consider adding further exceptions but I think these are the key ones for now.